### ISSUE TITLE
Sampling result in blurry or random results

### ISSUE THREAD

**Issue by XCZhang03** [alexzhou907/DDBM#8]
I was sampling with the e2h checkpoint and using the original script. However, I result in blurry or even random images. 
![sample_0](https://github.com/alexzhou907/DDBM/assets/97445569/100dbeeb-6331-4c94-b68f-412eaa524bd3)

**Comment by IqraNosheen786** [comment]
Hi @MaydaY-Tsinghua are you trying this code on your own dataset?

**Comment by XCZhang03** [comment]
No, I am using the original e2h data. However, the diode task works fine with the corresponding data and checkpoint.

**Comment by SDD2021-car** [comment]
Have you resolved this issue now? What changes need to be made in the parameter settings? Thanks @MaydaY-Tsinghua

**Comment by David-cripto** [comment]
Have you resolved this issue now? @SDD2021-car , @MaydaY-Tsinghua

**Comment by SDD2021-car** [comment]
> Have you resolved this issue now? [@SDD2021-car](https://github.com/SDD2021-car) , @MaydaY-Tsinghua

I’ve already resolved this issue. I believe it was caused by improper training parameter settings, not using the ODE-based method, or an insufficient number of training iterations. You can try training for 200,000 epochs while ensuring that the parameter settings are aligned with those in the original paper.

**Comment by yingli0505** [comment]
> No, I am using the original e2h data. However, the diode task works fine with the corresponding data and checkpoint.

hi, I am having the same issue. How did you fixed it? @XCZhang03
